webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -60,7 +60,9 @@ class ManageManifestModal extends Component {
   };
 
   saveOrganization = () => {
-    this.props.saveOrganization({ redhat_repository_url: this.state.redhat_repository_url });
+    // TODO: Necessary because of https://projects.theforeman.org/issues/26420
+    this.props.saveOrganization({ redhat_repository_url: this.state.redhat_repository_url })
+      .then(this.props.loadOrganization);
```

Comment:
@amirfefer @sharvit your thoughts on this approach? I don't know what else to do in this case.

Reply:
Can't we fix the server so it will return a valid response after put?

imo calling `loadOrganization` after `saveOrganization` is fine.
should it be done in the action `saveOrganization` itself since it produces a wrong state?

Reply:
We should fix the API, I filed a bug about it but I don't have enough expertise to fix it.   I will try to move this to the `saveOrganization` action, good point :+1: